example.env

```diff
@@ -3,3 +3,12 @@ NETWORK=<mainnet/goerli>
 VRF_COORDINATOR=<chainlink vrf>
 SUBSCRIPTION_ID=<chainlink>
 KEY_HASH=<chainlink key hash>
+
+# Rescue Trainer Purpose
```

Comment:
Need a README with detail descriptions.  If Kenneth needs to run it, he has to install node, hardhat, etc. on his machine, right?